contrib/hamilton/contrib/user/example_dataflow_template/dataflow_template/__init__.py

```diff
@@ -1,11 +1,11 @@
-# --- START LICENSE (optional)
-# --- END LICENSE
+# --- START NOTICES (optional)
+# --- END NOTICES
 # --- START IMPORT SECTION
 import logging
 
 logger = logging.getLogger(__name__)
 
-from hamilton import contrib
+from hamilton import contrib  # noqa E402
```

Comment:
what's the noqa for? I don't get any warning for it.

Reply:
Me messing around, I don't think its this